docs/conf.py

```diff
@@ -123,6 +123,7 @@
     # Files included in other rst files.
     'code-of-conduct.rst',
     'rules/periphery-rules.rst',
+    '**/cell-list.rst',
```

Comment:
I added this change to get rid of compilation warnings. There were 5 of them, like this:
```
/home/pzierhoffer/skywater-pdk/docs/contents/libraries/sky130_fd_sc_hdll/cell-list.rst:2: WARNING: duplicate label list of cells, other instance in /home/pzierhoffer/skywater-pdk/docs/contents/libraries/sky130_fd_sc_hdll/README.rst
```

What is strange is that I still get this warning, in two instances:
```
contents/libraries/sky130_fd_sc_hdll/cell-list.rst:2: WARNING: duplicate label list of cells, other instance in /home/pzierhoffer/skywater-pdk/docs/contents/libraries/sky130_fd_sc_hd/README.rst
contents/libraries/sky130_fd_sc_ls/cell-list.rst:2: WARNING: duplicate label list of cells, other instance in /home/pzierhoffer/skywater-pdk/docs/contents/libraries/sky130_fd_sc_hs/README.rst
```

No idea why. For some reason the first path is relative. Any ideas @mithro?

Reply:
README.rst including these files are identical to the others that do not warn.

Reply:
It's complaining because identical headers are found in multiple files and just reporting the first few it finds?

Reply:
weird that the result changes like that. But it does seem to be the case, thanks